assets/js/modules/adsense/datastore/urlchannels.js

```diff
@@ -114,7 +114,10 @@ const baseResolvers = {
 			return;
 		}
 
-		yield fetchGetURLChannelsStore.actions.fetchGetURLChannels( accountID, clientID );
+		const { error } = yield fetchGetURLChannelsStore.actions.fetchGetURLChannels( accountID, clientID );
+		if ( error ) {
+			yield errorStoreActions.receiveError( error, 'getURLChannels', [ accountID, clientID ] );
+		}
```

Comment:
@eugene-manuilov Isn't this already taken care of by `fetchGetURLChannels` via `createFetchStore`? Why is the extra `receiveError` call needed here? (This was already in the IB, but just struck me while reviewing here.)

Reply:
@felixarntz ah... yes, you are right. `createFetchStore` handles it. I forgot that it uses the baseName to save errors.